### DISCUSSION TITLE
How to disable the display of map components? (Zoom buttons, pin info window etc.)

### DISCUSSION THREAD

**Discussion by avukalov** [VladislavAntonyuk/MauiSamples#338] [Q&A] [answered]
Hello Vladislav, I noticed that when I click on a pin, in the bottom right corner of the map, it shows the direction icon and a Google Maps icon that opens the pin using the Google Maps app. I want to display a plain Google Map without any built-in components. Is it possible to achieve this? (Currently working on android but I need same on iOS if there is any of such components)

**Comment by VladislavAntonyuk** [comment, marked as answer]
Hello Antonio. You can disable zoom buttons (HasZoomEnabled), but it won't allow you zooming. Similar properties for other features: Has...Enabled. To disable window click, you can subscribe to InfoWindowClicked event and set HideInfoWindow = true.

On iOS, maps uses Apple Maps, not google maps.

**Reply by avukalov** [reply]
Vladislav, thank you for quick response :) I need a bottom sheet anyway, so it could be a workaround to keep zooming available and hide zoom buttons.
